### ISSUE TITLE
You are calling a deprecated version of the Ads API. Please update to the latest version: v2.12.

### ISSUE THREAD

**Issue by vaidotas-adf** [facebook/facebook-php-ads-sdk#411]
Latest facebook-php-ads-sdk version is 2.11.2. Should I just wait for update?

**Comment by windsfantasy6** [comment]
Could you tell us what endpoint were you trying to access?

**Comment by vaidotas-adf** [comment]
I'm trying to create ad preview using first example from this page: https://developers.facebook.com/docs/marketing-api/generatepreview/v2.12#examples
Before that I init app using `Api::init($app_id, $app_secret, $access_token);`

**Comment by poms32** [comment]
I have the same error : `You are calling a deprecated version of the Ads API. Please update to the latest version: v2.12.`

My request is pretty simple : 
`Api::init("XXXX", "XXXXXX", "XXXXXXX");
        $api = Api::instance();

        $account = new AdAccount("act_99999999999");
        dd($account->read());`

**Comment by suhaboncukcu** [comment]
I got the same error. Even when I try to use `dev-master` I still have it.

**Comment by windsfantasy6** [comment]
If you see this warning with an unpublished version, you can just ignore it.

**Comment by suhaboncukcu** [comment]
it throws an exception, so you can't really ignore it.

**I've figured that this is about graph_api version.** 

I first set `default_graph_version` for Facebook superclass and wherever applicable. But there were still some functions throwing the same exception.  I converted everything related to MarketingApi to rest version. (using curl examples)

**Comment by poms32** [comment]
Yes actually it drops an `AuthorizationException` so you can't do anything... an there is no possibility into Facebook dashboard to set older API version for apps

**Comment by SamCreamer** [comment]
Same problem. The sample code provided by Facebook throws an exception. Any way around this other than using cURL instead of the SDK?

**Comment by suhaboncukcu** [comment]
@SamCreamer I guess you can always fork the repo and change the version under `src/FacebookAds/ApiConfig.php` :)

**Comment by SamCreamer** [comment]
@suhaboncukcu That's a good point. I actually will do that. Will they merge my PR if I open one? Or is this issue intentional?

**Comment by suhaboncukcu** [comment]
@SamCreamer since I though it is something basic, I won't send a PR. I think they would fix it with some other things in a few days.
